op-node/rollup/derive/l1_block_info.go

```diff
@@ -259,6 +262,34 @@ func L1BlockInfoFromBytes(rollupCfg *rollup.Config, l2BlockTime uint64, data []b
 	return &info, info.unmarshalBinaryBedrock(data)
 }
 
+func (info *L1BlockInfo) marshalDepositsComplete() ([]byte, error) {
+	w := bytes.NewBuffer(make([]byte, 0, DepositsCompleteLen))
+	if err := solabi.WriteSignature(w, []byte(DepositsCompleteBytes4)); err != nil {
+		return nil, err
+	}
+	return w.Bytes(), nil
+}
+
+func (info *L1BlockInfo) unmarshalDepositsComplete(data []byte) error {
+	if len(data) != DepositsCompleteLen {
+		return fmt.Errorf("data is unexpected length: %d", len(data))
+	}
+	r := bytes.NewReader(data)
+
+	if _, err := solabi.ReadAndValidateSignature(r, DepositsCompleteBytes4); err != nil {
+		return err
+	}
+	if !solabi.EmptyReader(r) {
+		return errors.New("too many bytes")
+	}
+	return nil
+}
+
+func DepositCompletedFromBytes(rollupCfg *rollup.Config, l2BlockTime uint64, data []byte) (*L1BlockInfo, error) {
```

Comment:
Where is this function used? I don't understand what it is for and why it takes so many args that are unused

Reply:
similarly to the other ones, it should be used on the not-yet-existing tests. 
I can just remove it if it'd be cleaner